utils/oscap-xccdf.c

```diff
@@ -297,7 +297,7 @@ static struct oscap_module* XCCDF_SUBMODULES[] = {
     &XCCDF_VALIDATE_XML,
     &XCCDF_EXPORT_OVAL_VARIABLES,
     &XCCDF_GENERATE,
-	&XCCDF_REMEDIATE,
+    &XCCDF_REMEDIATE,
```

Comment:
Please undo this indentation change. Our convention is tabs for indents. We won't reindent the whole codebase to avoid conflicts but in new code we follow our convention.